docs/pages/environment_creation/index.md

```diff
@@ -5,116 +5,329 @@ title: Environment Creation
+            self.clock = pygame.time.Clock()
+
+        canvas = pygame.Surface((self.window_size, self.window_size))
+        canvas.fill((255, 255, 255))
+        pix_square_size = self.window_size / self.size      # The size of a single grid square in pixels
+
+        # First we draw the target
+        pygame.draw.rect(
+            canvas,
+            (255, 0, 0),
+            pygame.Rect(
+                pix_square_size * self._target_location,
+                (pix_square_size, pix_square_size),
+            ),
+        )
+        # Now we draw the agent
+        pygame.draw.circle(
+            canvas,
+            (0, 0, 255),
+            (self._agent_location + 0.5) * pix_square_size,
+            pix_square_size / 3,
+        )
+
+        # Finally, add some gridlines
+        for x in range(self.size + 1):
+            pygame.draw.line(
+                canvas,
+                0,
+                (0, pix_square_size * x),
+                (self.window_size, pix_square_size * x),
+                width=3,
+            )
+            pygame.draw.line(
+                canvas,
+                0,
+                (pix_square_size * x, 0),
+                (pix_square_size * x, self.window_size),
+                width=3,
+            )
+
+        if mode == "human":
+            # The following line copies our drawings from `canvas` to the visible window
+            self.window.blit(canvas, canvas.get_rect())
+            pygame.event.pump()
+            pygame.display.update()
+            
+            # We need to ensure that human-rendering occurs at the predefined framerate.
+            # The following line will automatically add a delay to keep the framerate stable.
+            self.clock.tick(self.metadata["render_fps"])
+        else:  # rgb_array
+            return np.transpose(
+                np.array(pygame.surfarray.pixels3d(canvas)), axes=(1, 0, 2)
+            )
+```
+
+### Close
+The `close` method should close any open resources that were used by the environment. In many cases,
+you don't actually have to bother to implement this method. However, in our example `render` may have
+been called with `mode="human"` and we might need to close the window that has been opened:
+
+```python
     def close(self):
-        # TODO
-        ...
+        if self.window is not None:
+            pygame.display.quit()
+            pygame.quit()
 ```
 
+In other environments `close`might also close files that were opened
+or release other resources. You shouldn't interact with the environment after having called `close`.
+
+
 ## Registering Envs
 
-In order for the custom environments to be detected by OpenAI gym, they must be registered as follows. We will choose to put this code in `gym-foo/gym_foo/__init__.py`. 
+In order for the custom environments to be detected by OpenAI gym, they must be registered as follows. We will choose to put this code in `gym-examples/gym_examples/__init__.py`. 
```

Comment:
Imo keep the naming consistent, so always write `Gym` instead of `OpenAI gym` (this applies to the entire file)